rest/client.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"crypto/hmac"
 	"crypto/sha256"
+	_ "crypto/sha512"
```

Comment:
I'll leave @kouno to respond, but I am pretty sure this relates to a very odd issue we have with our Comodo certificate, we should add a comment if this is the case.